examples/README.md

```diff
@@ -2,17 +2,59 @@
 
 Here we have put together a hand full of python examples to help get you started. As always feel free to [contact us](https://lob.com/support) directly if you have any questions on implementation.
 
+## Getting started
+Before running these examples make sure to do the following:
 
-## csv_address_verification
+1. Install the lob package with pip or easy_install:
```

Comment:
So what he's saying is that we should be referencing the local code rather than importing lob from pip. This prevents any issues with their installed version differing from the downloaded version. So the examples will always work.